apps/client/src/Components/CoursesPresentation/index.test.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import render from "Utils/render";
+
+import CoursesPresentation from ".";
+
+it("renders CoursesPresentation component", () => {
+  expect(() => render(<CoursesPresentation />)).not.toThrow();
+});
+
+describe("Check content of CoursesPresentation component", () => {
+  const { getByText } = render(<CoursesPresentation />);
+
+  it("checks if title rendered", () => {
+    const title = getByText("Lorem Ipsum");
```

Comment:
o Lorem Ipsum não deve ser hardcoded. o texto do componente deve:

- ser recebido da API, se for o componente que cuida da lógica de conexão com API e gerenciamento do estado
- ser extraído dos parâmetros que o componente recebe, se for um componente de estilização e display.

o lorem ipsum aqui do teste e do CoursesPresentation/index.tsx devem ser removidos

Reply:
Dúvida 1: O texto de toda a homepage vai vir da api?
Dúvida 2: Eu deixo o componente sem texto ou coloco o lorem ipsum na homepage e passo como parâmetro?

Reply:
1: sim, todos os textos vão necessariamente vir do strapi, onde eles vão ser inseridos por um usuário admin "criador de conteúdo". nada de conteúdo do site vai ser hardcoded

2: eu sugiro fazer como em src/Routes/Home/HomePage, e ter um componente filho Display que recebe o texto como parâmetro, e um componente pai que cuida da lógica de chamar a api e gerenciar o estado (por meio do hook de fetching do strapi) e entrega o texto pro Display.

aí nos testes vc dá pro display o texto que vc quiser como parâmetro